app/views/layouts/_service_notification_banner.html.erb

```diff
@@ -0,0 +1,12 @@
+<% unless @service_notifications.empty? %>
+  <% @service_notifications.each do |service_alert| %>
+      <div class="govuk-width-container app-site-width-container">
+        <div class="service_banner">
+          <button onclick="cancelThisNotification(this)" type="button" class="close-service-notification" id=<%= "ServiceNotifications." + service_alert['id'] %>>
```

Comment:
Only a tiny thing, but the `id=""` HTML attribute is missing the quote marks here.

We're getting HTML output like this at the moment:

```html
id=ServiceNotifications.ollie-test>
```

Where really we should have:

```html
id="ServiceNotifications.ollie-test">
```

Browsers are usually pretty chill about this stuff, and it's technically still valid HTML 5 without the quotes. But just to be safe, it's better to have them.